frontend/types/redemptionState.ts

```diff
@@ -0,0 +1,16 @@
+import { Nullable } from './utils';
+
+export enum RedemptionStep {
+  SelectCharity = 'SelectCharity',
+  SelectAmount = 'SelectAmount',
+  VerifyRedemption = 'VerifyRedemption',
```

Comment:
Shall we use UPPER_SNAKE_CASE for the enums?
```suggestion
  SELECT_CHARITY = 'SELECT_CHARITY',
  ...
```